filebeat/reader/multiline/multiline.go

```diff
@@ -48,6 +49,7 @@ type Reader struct {
 	separator    []byte
 	last         []byte
 	numLines     int
+	truncated    int
```

Comment:
Oh, I didn't think about multiline being able to capture truncated bytes as well. Good catch 👍